README.rst

```diff
@@ -10,3 +10,22 @@ Coaster: common patterns for Flask apps
 
 Coaster contains functions and db models for recurring patterns in Flask
 apps. Documentation at http://coaster.readthedocs.org/
+
+### Run tests
+
+Create a test Coaster DB:
+
+   $ createuser coaster
+   $ createdb -O coaster coaster_test
```

Comment:
I just confirmed that `tests/test_models.py` does not specify `coaster` as the username in the database connection. It defaults to the currently logged in user (see `whoami`) and Travis sets this as well, so the instructions should be amended accordingly.

```shell
$ createuser `whoami`
$ createdb -O `whoami` coaster_test
```